homeassistant/components/discord/notify.py

```diff
@@ -59,10 +59,23 @@ async def async_send_message(self, message, **kwargs):
 
         data = kwargs.get(ATTR_DATA) or {}
 
+        embed = None
         if ATTR_EMBED in data:
             embedding = data[ATTR_EMBED]
             fields = embedding.get(ATTR_EMBED_FIELDS)
 
+            if embedding:
+                embed = discord.Embed(**embedding)
+                if fields:
+                    for field_num, _ in enumerate(fields):
+                        embed.add_field(**fields[field_num])
```

Comment:
why not just
```
for field in fields:
    embed.add_field(**field)
```
?

Reply:
Good point, I was mainly just changing the existing code so didn't think about it (although these two lines did feel really odd)